deploy/rbd/helm/templates/NOTES.txt

```diff
@@ -0,0 +1 @@
+The Ceph RBD Container Storage Interface has been deployed.
```

Comment:
what is this for?

Reply:
This file is printed to the screen when the chart is installed by helm.  As an example, the metrics server chart prints how to use the metrics server data after you install it https://github.com/helm/charts/blob/master/stable/metrics-server/templates/NOTES.txt.  I think this could be a place to stick information on how to use this chart, maybe a link to the examples section of this repo?